src/components/Reaction/ReactionPickerWrapper.tsx

```diff
@@ -118,8 +144,19 @@ const ReactionPickerWrapper: React.FC<Props> = (props) => {
         handleReaction={handleReaction}
         hideReactionCount={hideReactionCount}
         hideReactionOwners={hideReactionOwners}
-        latestReactions={message.latest_reactions as LatestReactions}
-        reactionCounts={message.reaction_counts}
+        latestReactions={
+          message.latest_reactions as LatestReactions<At, Ch, Co, Me, Re, Us>
+        }
+        reactionCounts={
+          message.reaction_counts as MessageWithDates<
```

Comment:
if message is already `message: MessageWithDates<At, Ch, Co, Me, Re, Us>;`, do we need the casting?